src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -1670,7 +1232,7 @@ static int parse_args(std::vector<const char*>& args,
       cfg->remap_failure_fatal = true;
     } else if (ceph_argparse_flag(args, i, "--adapter-monitoring-enabled", (char *)NULL)) {
       cfg->adapter_monitoring_enabled = true;
-    } else if (ceph_argparse_witharg(args, i, &cfg->parent_pipe, err,
+    } else if (ceph_argparse_witharg(args, i, &parent_pipe, err,
```

Comment:
Might want to print that this option is deprecated and the value is ignored in the branch below.